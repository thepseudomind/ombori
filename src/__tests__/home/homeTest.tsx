import React from 'react';
import { shallow, render } from  'enzyme';
import Home from '../../pages/home/home';

// Test if homepage renders
describe('homepage test', ()=>{
    it('render page', ()=>{
        const wrapper = shallow(<Home/>);
        expect(wrapper).toMatchSnapshot();
    });
});
