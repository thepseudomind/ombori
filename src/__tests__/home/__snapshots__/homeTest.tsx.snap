// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`homepage test render page 1`] = `
<main
  className="home"
>
  <Main
    status={false}
    toggleSidebar={[Function]}
  />
  <Sidebar
    status={false}
    toggleSidebar={[Function]}
  />
</main>
`;
